### DISCUSSION TITLE
Error when implement the installing firebase_messaging: ^7.0.3

### DISCUSSION THREAD

**Discussion by Jesus180Reyes** [firebase/flutterfire#5104] [Q&A]
when i've clicked on pub get i have this message

flutter

`Because firebase_messaging >=7.0.3 <8.0.0-dev.7 depends on firebase_core ^0.5.0+1 and drivers_app depends on firebase_core ^0.7.0, firebase_messaging >=7.0.3 <8.0.0-dev.7 is forbidden.
So, because drivers_app depends on firebase_messaging ^7.0.3, version solving failed.
pub get failed (1; So, because drivers_app depends on firebase_messaging ^7.0.3, version solving failed.)`

**Comment by daadu** [comment]
The latest working firebase_messaging according to FlutterFire Roadmap [#2582] (which should work with `firebase_core ^0.7.0`) is in dev channel, check https://github.com/FirebaseExtended/flutterfire/discussions/4023.

Either wait for a stable `8.X.X` release that works with `firebase_core ^0.7.0` or point to the dev channel version `firebase_messaging: ^8.0.0-dev.15`
